include/database.py

```diff
@@ -59,3 +59,69 @@ def insert_returning(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+def remove_record(connectionString, sql, parameters):
+    try:
+        # Open the connection, override any timeout provided with something shorter
+        # since we expect to be running interactively
+        connection = psycopg2.connect(connectionString, connect_timeout=1)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, (parameters['id'],))
+
+        # Clean-up and return
+        connection.commit()
+        # check if the study exists
+        if cursor.rowcount:
+            print("Study deleted successfully")
+        else:
+            print("Study does not exist")
+
+        cursor.close()
+
+    except psycopg2.OperationalError as err:
+        sys.stderr.write(f'An error occurred connecting to the database: {err}')
+        raise DatabaseError
+
+    except psycopg2.DatabaseError as err:
+        sys.stderr.write(f'A general database error occurred: {err}')
+        raise DatabaseError
+
+    except psycopg2.Error as e:
+        print(type(e))
+        print(e)
+
+
+def rename_record(connectionString, sql, parameters):
```

Comment:
Separating out the `delete` and `update` operations is good, but having a function that only exists to rename records in a library for generic database operations is too specific. would be better to just rename this `update` and adjust the internals accordingly (mostly the same comments as above).

If this was a class that we were working in then an additional layer of abstraction would be to move most of the function into a function that _modifies_ data since a lot of the code is the same between each of the actions. Then expose `update`, `insert`, and `delete` methods since that provides better self-documenting code (e.g., `database.update` versus `database.execute_query`)